### PR TITLE
refactor(samples): fix samples format

### LINKED ISSUE #231: Fix samples format
Some of the samples migrated from GoogleCloudPlatform/java-docs-samples do not follow the samples guidelines.

### PR DESCRIPTION
**Pull request by suraj-qlogic**
Fixes #231

### CODE REVIEW

File: samples/snippets/src/main/java/com/example/monitoring/CreateAlertPolicy.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+  public static void createAlertPolicy(String projectId, String alertPolicyName)
+      throws IOException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests.
+    try (AlertPolicyServiceClient alertPolicyServiceClient = AlertPolicyServiceClient.create()) {
+      ProjectName name = ProjectName.of(projectId);
+
+      // A Filter that identifies which time series should be compared with the threshold
+      String metricFilter =
```
Comment (lesv) [inline samples/snippets/src/main/java/com/example/monitoring/CreateAlertPolicy.java:47]: Would this be better using string format? or some other way to not be quoting so much?

File: samples/snippets/src/main/java/com/example/monitoring/CreateAlertPolicy.java [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+              .build();
+
+      // Create an alert policy
+      AlertPolicy actualAlertPolicy = alertPolicyServiceClient.createAlertPolicy(name, alertPolicy);
+      String alertPolicyId =
+          actualAlertPolicy.getName().substring(actualAlertPolicy.getName().lastIndexOf("/") + 1);
+      System.out.println("alert policy created:" + alertPolicyId);
+    } catch (ApiException ex) {
+      System.out.print("\nalert policy was not created." + ex.toString());
+    }
```
Comment (lesv) [inline samples/snippets/src/main/java/com/example/monitoring/CreateAlertPolicy.java:93]: \n as string constants get picked up by static analyzers.
If you aren't doing anything with an exception, you probably shouldn't be catching it.  
See our [sample format guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md#exception-handling) for more info.

File: samples/snippets/src/main/java/com/example/monitoring/CreateMetricDescriptor.java [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+              .build();
+
+      // Send the request to create the metric descriptor
+      MetricDescriptor metricDescriptor = metricServiceClient.createMetricDescriptor(request);
+
+      // Process the response
+      System.out.println("metric descriptor created successfully: " + metricDescriptor.getName());
+    } catch (ApiException ex) {
+      System.out.print("\nmetric descriptor was not created." + ex.toString());
+    }
```
Comment (lesv) [inline samples/snippets/src/main/java/com/example/monitoring/CreateMetricDescriptor.java:73]: \n as string constants get picked up by static analyzers.
If you aren't doing anything with an exception, you probably shouldn't be catching it.  
See our [sample format guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md#exception-handling) for more info.

File: samples/snippets/src/main/java/com/example/monitoring/CreateTimeSeries.java [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+              .setName(ProjectName.of(projectId).toString())
+              .addAllTimeSeries(timeSeriesList)
+              .build();
+
+      // Writes time series data
+      metricServiceClient.createTimeSeries(request);
+      System.out.println("Done writing time series value");
+    } catch (ApiException ex) {
+      System.out.print("\n time series could not be written." + ex.toString());
+    }
```
Comment (lesv) [inline samples/snippets/src/main/java/com/example/monitoring/CreateTimeSeries.java:101]: \n as string constants get picked up by static analyzers.
If you aren't doing anything with an exception, you probably shouldn't be catching it.  
See our [sample format guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md#exception-handling) for more info.

File: samples/snippets/src/main/java/com/example/monitoring/DeleteAlertPolicy.java [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+  public static void deleteAlertPolicy(String alertPolicyId) throws IOException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests.
+    try (AlertPolicyServiceClient alertPolicyServiceClient = AlertPolicyServiceClient.create()) {
+      // Deletes an alerting policy
+      alertPolicyServiceClient.deleteAlertPolicy(alertPolicyId);
+      System.out.print("\nalert policy deleted successfully");
+    } catch (ApiException ex) {
+      System.out.print("\nalert policy not found:" + ex.toString());
+    }
```
Comment (lesv) [inline samples/snippets/src/main/java/com/example/monitoring/DeleteAlertPolicy.java:43]: \n as string constants get picked up by static analyzers.
If you aren't doing anything with an exception, you probably shouldn't be catching it.  
See our [sample format guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md#exception-handling) for more info.

File: samples/snippets/src/main/java/com/example/monitoring/DeleteMetricDescriptor.java [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests.
+    try (MetricServiceClient metricServiceClient = MetricServiceClient.create()) {
+      MetricDescriptorName metricName = MetricDescriptorName.of(projectId, name);
+      // Deletes a metric descriptor
+      metricServiceClient.deleteMetricDescriptor(metricName);
+      System.out.println("metric descriptor deleted successfully" + name);
+    } catch (ApiException ex) {
+      System.out.print("\nmetric descriptor was not deleted." + ex.toString());
+    }
```
Comment (lesv) [inline samples/snippets/src/main/java/com/example/monitoring/DeleteMetricDescriptor.java:45]: \n as string constants get picked up by static analyzers.
If you aren't doing anything with an exception, you probably shouldn't be catching it.  
See our [sample format guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md#exception-handling) for more info.

File: samples/snippets/src/main/java/com/example/monitoring/EnableDisableAlertPolicy.java [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+
+  public static void enableDisableAlertPolicy(
+      String alertPolicyId, boolean status, String displayName) throws IOException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests.
+    try (AlertPolicyServiceClient alertPolicyServiceClient = AlertPolicyServiceClient.create()) {
+      // A Filter that identifies which time series should be compared with the threshold
+      String metricFilter =
+          "metric.type=\"compute.googleapis.com/instance/"
+              + "cpu/utilization\" AND resource.type=\"gce_instance\"";
```
Comment (lesv) [inline samples/snippets/src/main/java/com/example/monitoring/EnableDisableAlertPolicy.java:49]: All this quoting is difficult to follow, perhaps there is a better way to express this?  String.Format or Stirng.Join combined w/ String.Format ??

File: samples/snippets/src/main/java/com/example/monitoring/EnableDisableAlertPolicy.java [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+          UpdateAlertPolicyRequest.newBuilder().setAlertPolicy(alertPolicy).build();
+
+      // Process update policy request
+      AlertPolicy updatedAlertPolicy =
+          alertPolicyServiceClient.updateAlertPolicy(updateAlertPolicyRequest);
+      System.out.print(
+          "alert policy enable disable status:" + updatedAlertPolicy.getEnabled() + "\n");
+    } catch (ApiException ex) {
+      System.out.print("alert policy not found:" + ex.toString());
+    }
```
Comment (lesv) [inline samples/snippets/src/main/java/com/example/monitoring/EnableDisableAlertPolicy.java:102]: If you aren't doing anything with an exception, you probably shouldn't be catching it.  
See our [sample format guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md#exception-handling) for more info.

File: samples/snippets/src/main/java/com/example/monitoring/GetAlertPolicy.java [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+  public static void getAlertPolicy(String alertPolicyName) throws IOException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests.
+    try (AlertPolicyServiceClient alertPolicyServiceClient = AlertPolicyServiceClient.create()) {
+      // Gets a single alerting policy
+      AlertPolicy alertPolicy = alertPolicyServiceClient.getAlertPolicy(alertPolicyName);
+      System.out.print("alert policy retrieved successfully:" + alertPolicy.getName() + "\n");
+    } catch (ApiException ex) {
+      System.out.print("alert policy not found:" + ex.toString());
+    }
```
Comment (lesv) [inline samples/snippets/src/main/java/com/example/monitoring/GetAlertPolicy.java:43]: If you aren't doing anything with an exception, you probably shouldn't be catching it.  
See our [sample format guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md#exception-handling) for more info.

File: samples/snippets/src/main/java/com/example/monitoring/ListTimeSeries.java [hunk 10]
```diff
@@ -0,0 +1,72 @@
...
+      // Send the request to list the time series
+      MetricServiceClient.ListTimeSeriesPagedResponse response =
+          metricServiceClient.listTimeSeries(request);
+
+      // Process the response
+      System.out.println("Got timeseries: ");
+      response.iterateAll().forEach(timeSeries -> System.out.println(timeSeries));
+    } catch (ApiException ex) {
+      System.out.print("\n time series not found." + ex.toString());
+    }
```
Comment (lesv) [inline samples/snippets/src/main/java/com/example/monitoring/ListTimeSeries.java:69]: \n as string constants get picked up by static analyzers.
If you aren't doing anything with an exception, you probably shouldn't be catching it.  
See our [sample format guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md#exception-handling) for more info.

File: samples/snippets/src/main/java/com/example/monitoring/MonitoredResourcesList.java [hunk 11]
```diff
@@ -0,0 +1,62 @@
...
+      System.out.println("Listing monitored resource descriptors: ");
+      response
+          .iterateAll()
+          .forEach(
+              metricDescriptor ->
+                  System.out.printf(
+                      "success! monitored resources type %s \n", metricDescriptor.getType()));
+    } catch (ApiException ex) {
+      System.out.print("\nmonitored resource descriptors not found." + ex.toString());
+    }
```
Comment (lesv) [inline samples/snippets/src/main/java/com/example/monitoring/MonitoredResourcesList.java:59]: \n as string constants get picked up by static analyzers.
If you aren't doing anything with an exception, you probably shouldn't be catching it.  
See our [sample format guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md#exception-handling) for more info.

File: samples/snippets/src/main/java/com/example/monitoring/UpdateAlertPolicy.java [hunk 12]
```diff
@@ -0,0 +1,98 @@
...
+      UpdateAlertPolicyRequest updateAlertPolicyRequest =
+          UpdateAlertPolicyRequest.newBuilder().setAlertPolicy(alertPolicy).build();
+
+      // process update policy request
+      AlertPolicy updatedAlertPolicy =
+          alertPolicyServiceClient.updateAlertPolicy(updateAlertPolicyRequest);
+      System.out.print("alert policy updated successfully:" + updatedAlertPolicy.getDisplayName());
+    } catch (ApiException ex) {
+      System.out.print("alert policy not found:" + ex.toString());
+    }
```
Comment (lesv) [inline samples/snippets/src/main/java/com/example/monitoring/UpdateAlertPolicy.java:95]: If you aren't doing anything with an exception, you probably shouldn't be catching it.  
See our [sample format guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md#exception-handling) for more info.

File: samples/snippets/src/test/java/com/example/monitoring/CreateAlertPolicyIT.java [hunk 13]
```diff
@@ -0,0 +1,83 @@
...
+  }
+
+  @BeforeClass
+  public static void checkRequirements() {
+    requireEnvVar("GOOGLE_CLOUD_PROJECT");
+  }
+
+  @Before
+  public void setUp() {
+    alertPolicyDisplayName = "alert_policy_name_" + UUID.randomUUID().toString().substring(0, 8);
```
Comment (lesv) [inline samples/snippets/src/test/java/com/example/monitoring/CreateAlertPolicyIT.java:58]: Some bits in the first part of the uuid are constants.

**Review by lesv — CHANGES_REQUESTED** [review]

File: samples/snippets/src/main/java/com/example/monitoring/CreateAlertPolicy.java (hunk 1 above)
Comment (suraj-qlogic) [reply samples/snippets/src/main/java/com/example/monitoring/CreateAlertPolicy.java:47]: Done

File: samples/snippets/src/main/java/com/example/monitoring/CreateAlertPolicy.java (hunk 2 above)
Comment (suraj-qlogic) [reply samples/snippets/src/main/java/com/example/monitoring/CreateAlertPolicy.java]: Done

File: samples/snippets/src/main/java/com/example/monitoring/CreateMetricDescriptor.java (hunk 3 above)
Comment (suraj-qlogic) [reply samples/snippets/src/main/java/com/example/monitoring/CreateMetricDescriptor.java]: Done

File: samples/snippets/src/main/java/com/example/monitoring/CreateTimeSeries.java (hunk 4 above)
Comment (suraj-qlogic) [reply samples/snippets/src/main/java/com/example/monitoring/CreateTimeSeries.java]: Done

File: samples/snippets/src/main/java/com/example/monitoring/DeleteAlertPolicy.java (hunk 5 above)
Comment (suraj-qlogic) [reply samples/snippets/src/main/java/com/example/monitoring/DeleteAlertPolicy.java]: Done

File: samples/snippets/src/main/java/com/example/monitoring/ListTimeSeries.java (hunk 10 above)
Comment (suraj-qlogic) [reply samples/snippets/src/main/java/com/example/monitoring/ListTimeSeries.java]: Done

File: samples/snippets/src/main/java/com/example/monitoring/MonitoredResourcesList.java (hunk 11 above)
Comment (suraj-qlogic) [reply samples/snippets/src/main/java/com/example/monitoring/MonitoredResourcesList.java]: Done

File: samples/snippets/src/main/java/com/example/monitoring/TimeSeriesHeadersList.java [hunk 14]
```diff
@@ -0,0 +1,73 @@
...
+      // Send the request to list the time series
+      MetricServiceClient.ListTimeSeriesPagedResponse response =
+          metricServiceClient.listTimeSeries(request);
+
+      // Process the response
+      System.out.println("Got timeseries headers: ");
+      response.iterateAll().forEach(timeSeries -> System.out.println(timeSeries));
+    } catch (ApiException ex) {
+      System.out.print("\n time series not found." + ex.toString());
+    }
```
[the remark this answers is not in the document]
Comment (suraj-qlogic) [reply samples/snippets/src/main/java/com/example/monitoring/TimeSeriesHeadersList.java]: Done

File: samples/snippets/src/main/java/com/example/monitoring/UpdateAlertPolicy.java (hunk 12 above)
Comment (suraj-qlogic) [reply samples/snippets/src/main/java/com/example/monitoring/UpdateAlertPolicy.java]: Done

File: samples/snippets/src/test/java/com/example/monitoring/CreateAlertPolicyIT.java (hunk 13 above)
Comment (suraj-qlogic) [reply samples/snippets/src/test/java/com/example/monitoring/CreateAlertPolicyIT.java]: Done

File: samples/snippets/src/main/java/com/example/monitoring/DeleteMetricDescriptor.java (hunk 6 above)
Comment (suraj-qlogic) [reply samples/snippets/src/main/java/com/example/monitoring/DeleteMetricDescriptor.java]: Done

File: samples/snippets/src/main/java/com/example/monitoring/EnableDisableAlertPolicy.java (hunk 7 above)
Comment (suraj-qlogic) [reply samples/snippets/src/main/java/com/example/monitoring/EnableDisableAlertPolicy.java]: Done

File: samples/snippets/src/main/java/com/example/monitoring/EnableDisableAlertPolicy.java (hunk 8 above)
Comment (suraj-qlogic) [reply samples/snippets/src/main/java/com/example/monitoring/EnableDisableAlertPolicy.java]: Done

File: samples/snippets/src/main/java/com/example/monitoring/GetAlertPolicy.java (hunk 9 above)
Comment (suraj-qlogic) [reply samples/snippets/src/main/java/com/example/monitoring/GetAlertPolicy.java]: Done

File: samples/snippets/src/main/java/com/example/monitoring/GetMonitoredResource.java [hunk 15]
```diff
@@ -0,0 +1,52 @@
...
+      MonitoredResourceDescriptorName name =
+          MonitoredResourceDescriptorName.of(projectId, resourceId);
+
+      // Prepare get monitor resource descriptor
+      MonitoredResourceDescriptor response =
+          metricServiceClient.getMonitoredResourceDescriptor(name);
+      System.out.println(response);
+    } catch (ApiException ex) {
+      System.out.print("\nmonitored resource not found." + ex.toString());
+    }
```
[the remark this answers is not in the document]
Comment (suraj-qlogic) [reply samples/snippets/src/main/java/com/example/monitoring/GetMonitoredResource.java]: Done

File: samples/snippets/src/main/java/com/example/monitoring/ListAlertPolicy.java [hunk 16]
```diff
@@ -0,0 +1,62 @@
...
+      response
+          .iterateAll()
+          .forEach(
+              alertPolicy ->
+                  System.out.printf(
+                      "success! alert policy %s is policyId %s\n",
+                      alertPolicy.getDisplayName(), alertPolicy.getName()));
+    } catch (ApiException ex) {
+      System.out.print("alert policy not found:" + ex.toString());
+    }
```
[the remark this answers is not in the document]
Comment (suraj-qlogic) [reply samples/snippets/src/main/java/com/example/monitoring/ListAlertPolicy.java]: Done

File: samples/snippets/src/main/java/com/example/monitoring/ListMetricDescriptor.java [hunk 17]
```diff
@@ -0,0 +1,62 @@
...
+      response
+          .iterateAll()
+          .forEach(
+              metricDescriptor ->
+                  System.out.printf(
+                      "success! metric descriptor with name %s display name %s\n",
+                      metricDescriptor.getName(), metricDescriptor.getDisplayName()));
+    } catch (ApiException ex) {
+      System.out.print("\nmetric descriptor not found." + ex.toString());
+    }
```
[the remark this answers is not in the document]
Comment (suraj-qlogic) [reply samples/snippets/src/main/java/com/example/monitoring/ListMetricDescriptor.java]: Done

File: samples/snippets/src/main/java/com/example/monitoring/CreateAlertPolicy.java (hunk 1 above)
Comment (lesv) [reply samples/snippets/src/main/java/com/example/monitoring/CreateAlertPolicy.java:47]: Not quite what I was thinking.  Users are going to want to customize this and you have lots of `\"` embedded within a string.  The first thing I found when I Google'd was this idea: https://stackoverflow.com/questions/15305956/how-to-add-quotes-around-printed-string
```java
String metricFilter = "metric.type=" 
          + '"' 
          + "compute.googleapis.com/instance/cpu/utilization" 
          + '"' 
          + " AND resource.type="
          + '"'
          + "gce_instance"
          + '"';
```

What I'm asking for is that we make it more readable and easier for users to modify and adjust.

File: samples/snippets/src/main/java/com/example/monitoring/UpdateAlertPolicy.java [hunk 18]
```diff
@@ -0,0 +1,97 @@
...
+  public static void updateAlertPolicy(String alertPolicyName, String newPolicyName)
+      throws ApiException, IOException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests.
+    try (AlertPolicyServiceClient alertPolicyServiceClient = AlertPolicyServiceClient.create()) {
+      // A Filter that identifies which time series should be compared with the threshold
+      String metricFilter =
+          String.format(
+              "metric.type=\"compute.googleapis.com/instance/"
+                  + "cpu/utilization\" AND resource.type=\"gce_instance\"");
```
Comment (lesv) [inline samples/snippets/src/main/java/com/example/monitoring/UpdateAlertPolicy.java:48]: Same issue, can we make it easier to read and to edit for customers?

Trying a different approach this time.
```java
String metricFilter = 
        "metric.type="
        + '"' + compute.googleapis.com/instance/cpu/utilization" + '"'
        + " AND "
        + "resource.type="
        + '"' + "gce_instance" + '"";
```

**Review by lesv — CHANGES_REQUESTED** [review]
If all we were doing was executing this code I'd LGTM it, my two comments on string literals with quotes in them are about aesthetics and clarity - how can we make this easier to read and understand.   If you can come up with an idea that accomplishes that better than my suggestions - please try it.

File: samples/snippets/src/main/java/com/example/monitoring/UpdateAlertPolicy.java (hunk 18 above)
Comment (suraj-qlogic) [reply samples/snippets/src/main/java/com/example/monitoring/UpdateAlertPolicy.java]: Done

File: samples/snippets/src/main/java/com/example/monitoring/CreateAlertPolicy.java (hunk 1 above)
Comment (suraj-qlogic) [reply samples/snippets/src/main/java/com/example/monitoring/CreateAlertPolicy.java:49]: Done

**Review by lesv — APPROVED** [review]

**Comment by suraj-qlogic** [comment]
@chingor13 ,gentle ping

**Comment by suraj-qlogic** [comment]
@chingor13 , PTAL

**Pull request merged**